crates/bevy_window/src/window.rs

```diff
@@ -140,6 +142,11 @@ impl Window {
         self.height
     }
 
+    #[inline]
+    pub fn maximized(&self) -> bool {
+        self.maximized
+    }
+
     pub fn set_resolution(&mut self, width: u32, height: u32) {
```

Comment:
Can we add a `set_maximized(&mut self, maximized: bool)` function that produces a WindowCommand? It would be nice to let people maximize on-demand.